src/AbstractFixer.php

```diff
@@ -104,8 +110,12 @@ public function configure(array $configuration = null)
             $configuration = array();
         }
 
+        if (null === $this->configurationDefinition) {
+            $this->configurationDefinition = $this->getConfigurationDefinition();
```

Comment:
with this approach one will still call `getConfigurationDefitinion()` and create new object all the time.
it is getter that should provide cached object.